users/urls.py

```diff
@@ -2,15 +2,24 @@
 from django.urls import path
 
 from users.views import (
+    FollowerViewSet,
     RetrieveUpdateSelfUserProfileView,
     RetrieveUserProfileView,
     SearchUserView,
     SignupView,
 )
 
+set_follower = FollowerViewSet.as_view(
```

Comment:
Follower is someone you follow. Probably you mean `set-following` or `follow`

Reply:
Yes, renamed